packages/@aws-cdk/aws-stepfunctions-tasks/test/integ.call-dynamodb.ts

```diff
@@ -0,0 +1,94 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as tasks from '../lib';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn> : should return execution arn
+ * *
+ * * aws dynamodb scan --table-name Messages --query "Items": should return an empty array
+ * * aws stepfunctions describe-execution --execution-arn <exection-arn generated before> --query 'status': should return status as SUCCEEDED
```

Comment:
This won't quite work -

1. The table `Messages` does not exist. Add a line here above 'Stack verification steps' with the AWS CLI command to create this table.
2. A scan would return empty even if the step function did nothing. Instead, I suggest modifying this as so -
     - Put Item (random number)
     - Update Item (add another random number)
     - Get Item and store the retrieved as the output of the step function
     - Delete Item
  Add an AWS CLI command retrieves the output of the step function (as proof that it actually executed) to the verification steps.

Reply:
Will do